index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nylbert's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <div id="ham-menu">
+          <button id="ham-but" type="button" class="hamburger">
+            <span class="ham-top"></span>
+            <span class="ham-mid"></span>
+            <span class="ham-botom"></span>
+          </button>
+        </div>
+        <ul id="menu">
+          <il><a href="#portfolio">Portfolio</a></il>
+          <il><a href="#about">About</a></il>
+          <il><a href="#contact">Contact</a></il>
+        </ul>
+      </div>
+    </header>
+    <div id="center-div">
+        <div id="intro">
+          <h1 id="name" class="main-title">Hey there. I'm Nylbert.</h1>
+          <h1 id="profile" class="main-title">I'm a software developer.</h1>
+          <h3 id="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</h3>
+          <div id="sm-icons">
```

Comment:
- [ ] Please use a `ul` instead of a `div`, and wrap all the `a` tags in a `li`. This way, you can group your social icons. in an unordered list.
- [ ] Kindly add Medium and AngleList icons as well.